### PR TITLE
Furthermore enhance german translation

### PR DESCRIPTION
**Pull request by Lucki**
> I find all compound nouns with "Dokument...." are not written as "Dokumenten..." Sounds kind of weird to me, too. I would basically write e.g. "Dokumentname". [linguee.de/englisch-deutsch/uebersetzung/document+name.html](https://www.linguee.de/englisch-deutsch/uebersetzung/document+name.html)

@monnypython 
Docspell is all about documents - we could just leave out that part where it's obvious.

### CODE REVIEW

File: modules/webapp/src/main/elm/Messages/Comp/ItemCard.elm [hunk 1]
```diff
@@ -43,9 +43,9 @@ de =
     { basics = Messages.Basics.de
     , dueOn = "Fällig am"
     , new = "Neu"
-    , openAttachmentFile = "Öffne Anhang"
-    , gotoDetail = "Gehe zur Detailansicht"
-    , cycleAttachments = "Gehe durch Anhänge"
+    , openAttachmentFile = "Anhang öffnen"
+    , gotoDetail = "Detailansicht"
+    , cycleAttachments = "Anhänge durchschalten"
```
Comment (eikek) [inline modules/webapp/src/main/elm/Messages/Comp/ItemCard.elm:48]: This is hard to translate into German, imho :) tbh I wouldn't know what that means (but the former was not better at all)

File: modules/webapp/src/main/elm/Messages/Comp/SearchMenu.elm [hunk 2]
```diff
@@ -88,10 +88,10 @@ de =
     , switchSearchModes = "Zwischen den Such-Modi wechseln"
     , contentSearch = "Volltextsuche…"
     , searchInNamesPlaceholder = "Suche in Titeln…"
-    , fulltextSearchInfo = "Volltextsuche im Dokumenteninhalt und in Notizen."
+    , fulltextSearchInfo = "Volltextsuche im Inhalt und in Notizen."
     , nameSearchInfo = "Sucht in Namen von Korrespondent/Betreffend, Dokumenten und Notizen."
     , tagCategoryTab = "Tag-Kategorien"
-    , chooseOrganization = "Wähle eine Organisation"
+    , chooseOrganization = "Wähle ein Unternehmen"
```
Comment (eikek) [inline modules/webapp/src/main/elm/Messages/Comp/SearchMenu.elm:94]: This is tricky imho. Not all organizations are companies or enterprises etc. You might have universities, clubs, associations etc. wdyt?

**Comment by eikek** [comment]
It looks nicer, thank you. The only thing where I'm not convinced is that "Organisation" is replaced by "Unternehmen". "Unternehmen" means "company/enterprise", but you may have documents from your country authorities, clubs, universities etc. To me "Unternehmen" is too special. Are there other opinions?

File: modules/webapp/src/main/elm/Messages/Comp/SearchMenu.elm (hunk 2 above)
Comment (Lucki) [reply modules/webapp/src/main/elm/Messages/Comp/SearchMenu.elm:94]: I've gone alphabetically through the files and the first one already had Unternehmen so I've changed the other ones.

File: modules/webapp/src/main/elm/Messages/Comp/UserTable.elm [hunk 3]
```diff
@@ -40,5 +40,5 @@ de =
     , email = "E-Mail"
     , logins = "Anmeldungen"
     , lastLogin = "Letzte Anmeldung"
-    , formatDateTime = DF.formatDateTimeLong Messages.UiLanguage.English
+    , formatDateTime = DF.formatDateTimeLong Messages.UiLanguage.German
```
Comment (Lucki) [inline modules/webapp/src/main/elm/Messages/Comp/UserTable.elm:43]: Not sure if this was intentionally left english

File: modules/webapp/src/main/elm/Messages/Comp/SearchMenu.elm (hunk 2 above)
Comment (eikek) [reply modules/webapp/src/main/elm/Messages/Comp/SearchMenu.elm]: Oh this was a mistake then. Many thanks for changing it again!

File: modules/webapp/src/main/elm/Messages/Comp/ItemDetail/MultiEditMenu.elm [hunk 4]
```diff
@@ -53,8 +53,8 @@ de =
     , chooseDirection = "Wähle eine Richtung…"
     , confirmUnconfirm = "Bestätige/Widerrufe Metadaten"
     , confirm = "Bestätige"
-    , unconfirm = "Widerufe Betätigung"
-    , changeTagMode = "Wechsel den Änderungs-Modus für Tags"
-    , dueDateTab = "Fälligkeits-Datum"
+    , unconfirm = "Widerrufe Betätigung"
```
Comment (eikek) [inline modules/webapp/src/main/elm/Messages/Comp/ItemDetail/MultiEditMenu.elm:56]: Here the typo has made it through :-) "Betätigung" -> "Bestätigung"

File: modules/webapp/src/main/elm/Messages/Comp/UserTable.elm (hunk 3 above)
Comment (eikek) [reply modules/webapp/src/main/elm/Messages/Comp/UserTable.elm:43]: No, was a mistake. Good catch!

**Comment by eikek** [comment]
Thank you!

**Pull request merged**